### DISCUSSION TITLE
Remove owner of respository?

### DISCUSSION THREAD

**Discussion by ghost** [FlareSolverr/FlareSolverr#1422] [General]
Can we get the owner of the repository removed? Issues have gone unaddressed for 6 months and same with pull requests. It's sad to see such a useful tool be abandoned.

**Comment by ilike2burnthing** [comment]
It's not been abandoned - https://github.com/FlareSolverr/FlareSolverr/commits/master/ 

We're open to PRs, and there are a couple of contributor builds that many are using, however they aren't perfect.

**Comment by ghost** [comment]
No update since June, it's abandoned regardless of whatever commits have happened. Doesn't work on my PTs anymore at all but the contributor builds are definitely much better.

**Comment by ilike2burnthing** [comment]
Ah yes, tell the dev making the commits and replying to you within a few minutes that it's been abandoned 🙄

**Comment by ghost** [comment]
Ah yes, ignore the fact that there hasn't been a release since June, it doesn't work on PTs that it used to, and that it is functionally abandoned despite commits happening. It's like you don't comprehend English.

**Comment by ilike2burnthing** [comment]
It's almost like I said that we're open to PRs... something I've also said numerous times in other issues, discussions, and PRs. 

There hasn't been a new update because none of the recent commits have necessitated one (also because I know if I published a new release for them I'd be flooded with 'new version still no worky' posts from people who refuse to read).

The next release will be one that resolves these issues. The current PRs and contributor builds haven't been merged because they have their own issues (proxies not working, memory leaks, not compatible with our builder, etc.).

Is there at least one active dev? Yes. Does said dev engage with issues, discussions, and PRs? When I have time. Have there been recent commits? Yes. 

If you want to say that this project hasn't had any recent releases, and that the current official release doesn't work, that would be correct. But very obviously the project hasn't been abandoned; were someone to make a PR with no downsides, I'd merge and publish in a second, and continue maintaining the project.

**Comment by ghost** [comment]
Contributor builds work just fine and have fixed all of the issues I personally had, so you can keep "maintaining" this project I guess. I've moved on.

**Comment by ilike2burnthing** [comment]
Cool... Again, like I said, they have their own issues, hence not merging. I've been promoting the use of them to people who post here, I'm using one myself, but they're not suitable for official release. Glad I could clarify things for you.
